bodhi/server/config.py

```diff
@@ -336,33 +336,6 @@ class BodhiConfig(dict):
         'cache_dir': {
             'value': None,
             'validator': _validate_none_or(validate_path)},
-        'captcha.background_color': {
-            'value': '#ffffff',
-            'validator': _validate_color},
-        'captcha.font_color': {
-            'value': '#000000',
-            'validator': _validate_color},
-        'captcha.font_path': {
-            'value': '/usr/share/fonts/liberation/LiberationMono-Regular.ttf',
```

Comment:
I think we have the Ansible project installing this, and if so we can drop that too.